dist/s_funcs.list

```diff
@@ -33,6 +33,7 @@ __wt_stat_join_aggregate
 __wt_stat_join_clear_all
 __wt_stream_set_no_buffer
 __wt_try_readlock
+populate_thread
```

Comment:
Putting this function into the unused-function check skip list, as it's actually invoked as the argument of another function `start_threads`.